upgrades/README.md

```diff
@@ -137,3 +137,12 @@ Report:  /tmp/utf/test-results/pre-upgrade/report.html
 ```
 
 These files would be useful when a specific Test Action failed and the reason why isn't apparent from either the messaging to STDOUT or the UTF Run Log.  More information about what these files are can be found in [the Robot Framework's documentation](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#different-output-files).
+
+#### Current State & Limitations
+
+The UTF is currently functional and runnable, but of limited utility. The following are areas where we expect to need to invest more time or investigation:
+- Adding more expectations to the knowledge base and corresponding test actions. There is a discussion in #68 regarding the format of expectations. Generally, the value of this project is directly correlated to the number and breadth of the expectations and tests, so this is a very important area of investment.
```

Comment:
nit: You may want to like the PR to #68, I thought this may happen automatically but when I looked on your branch it didn't link. Lmk if I'm wrong here

Reply:
Ah, that's what I was hoping would happen too. Thanks, I'll do a real link.